src/wasmer.hh

```diff
@@ -1,17 +1,20 @@
+
 #if !defined(WASMER_H_MACROS)
-  #define WASMER_H_MACROS
-  #if defined(MSVC)
-    #if defined(_M_AMD64)
-      #define ARCH_X86_64
-    #endif
-  #endif
-
-#if defined(GCC) || defined(__clang__)
-    #if defined(__x86_64__)
-      #define ARCH_X86_64
-    #endif
-  #endif
+#define WASMER_H_MACROS
+
+#if defined(MSVC)
```

Comment:
looks indentation has been lost here. i dunno if that's important though. other than that +1.

Reply:
Actually, it's better. `#` is better at column 0. Some C compilers prefer that :-).